tests/src/test_helpers/ssl_helpers.c

```diff
@@ -22,6 +22,14 @@
 
 #include <test/ssl_helpers.h>
 
+/*
+ * MSVC would report warning (4267) of conversion from size_t to int.
+ * But this has been resolved for functions defined in this file.
+ */
+#if defined(_MSC_VER)
+#pragma warning( disable: 4267 )
+#endif /* _MSC_VER */
```

Comment:
Unless there's really no idiomatic way to write the code to avoid the warnings (e.g. there are some functions in the constant time module that perform unary minus on signed integers, and MSC really doesn't like that) it's always best to leave the warning on and change the code.

In this case, having the casts in the appropriate places (together with code that makes sure the casts are safe) is surely preferable to turning off the warning - i.e please don't turn off this warning

Reply:
This makes sense. Will change.